### PR TITLE
Clarify which logged airspeed value is used for transition.

### PR DESCRIPTION
**Pull request by timtuxworth**
When trying to figure out why a plane can't successfully transition, it can be confusing to figure out which log value to refer to to determine what the AP is using as the values can be different. This explicitly explains that it is CTUN.As that is used.

### CODE REVIEW

File: plane/source/docs/quadplane-transitions.rst [hunk 1]
```diff
@@ -25,7 +25,7 @@ Transition to Fixed Wing Mode from VTOL
 
 .. warning:: If you do not have sufficient airspeed, an immediate stall will occur! Since MANUAL mode is often setup as a reflex driven "bail-out", some users move, or remove this mode, and substitute QSTABLIZE or QLOITER as an alternative "bail-out" for a QuadPlane. Also if the transition is made downwind, the transition time is short, and no airspeed sensor is used in non-tailsitters, a stall could occur also since insufficient airspeed has been obtained when VTOL assistance has terminated.
  
--  If you transition to any other fixed wing mode then the VTOL motors will continue to supply lift and stability until you have reached the :ref:`ARSPD_FBW_MIN <ARSPD_FBW_MIN>` airspeed (or airspeed estimate if no airspeed sensor). This phase is called "Transition airspeed wait".
+-  If you transition to any other fixed wing mode then the VTOL motors will continue to supply lift and stability until you have reached the :ref:`ARSPD_FBW_MIN <ARSPD_FBW_MIN>` airspeed (or airspeed estimate if no airspeed sensor).  This phase is called "Transition airspeed wait".
```
Comment (Hwurzburg) [inline plane/source/docs/quadplane-transitions.rst:28]: extra whitespace added....remove

File: plane/source/docs/quadplane-transitions.rst [hunk 2]
```diff
@@ -39,6 +39,7 @@ Transition to Fixed Wing Mode from VTOL
 
 .. note:: if bit 19 of :ref:`Q_OPTIONS<Q_OPTIONS>` is set and :ref:`Q_TRANS_FAIL<Q_TRANS_FAIL>` is not zero, and if the airspeed is greater than 1/2 of :ref:`ARSPD_FBW_MIN<ARSPD_FBW_MIN>`, then the transition to fixed wing will immediately complete. This is useful if no airspeed sensor is being used, and the transition is into a headwind, which could prevent an accurate airspeed estimate from being obtained until a turn is made. Without using the :ref:`Q_TRANS_FAIL<Q_TRANS_FAIL>` timeout and this Q_OPTION, the transition could be indefinitely long since airspeed might be reported as below :ref:`ARSPD_FBW_MIN<ARSPD_FBW_MIN>` due to low groundspeed.
 
+.. note:: The airspeed used for transition can be found in dataflash logs as CTUN.As. This is a canonical value which will incude the airspeed sensor if enabled, or use the synthetic airspeed if not, so if you are having trouble acheiving the transition speed, this is the value you should refer to.
```
Comment (Hwurzburg) [inline plane/source/docs/quadplane-transitions.rst:42]:
```suggestion
.. note:: The airspeed used during transition can be found in dataflash logs as CTUN.As. This is a canonical value which will incudes the airspeed sensor if enabled, or use the synthetic airspeed if not, so if you are having trouble achieving the transition speed, this is the value you should refer to.
```

spelling and grammar

**Review by Hwurzburg — CHANGES_REQUESTED** [review]

File: plane/source/docs/quadplane-transitions.rst (hunk 1 above)
Comment (timtuxworth) [reply plane/source/docs/quadplane-transitions.rst:28]: fixed

File: plane/source/docs/quadplane-transitions.rst (hunk 2 above)
Comment (timtuxworth) [reply plane/source/docs/quadplane-transitions.rst:42]: I guess we both suffer from that. ("incudes" in your suggestion), I've corrected both, but also simplified the complex sentence at the end.

**Comment by timtuxworth** [comment]
How does this look @Hwurzburg ?

**Comment by Hwurzburg** [comment]
LGTM...thanks

**Pull request merged**